src/Traits/ReflectionClassLikeTrait.php

```diff
@@ -345,9 +346,9 @@ function (array &$result, \ReflectionClass $instance) {
 
     /**
      * {@inheritdoc}
-     * @param string $name
      */
-    public function getMethod($name)
+    #[\ReturnTypeWillChange]
```

Comment:
@lisachenko  this is needed due to original Reflection return `ReflectionMethod`, while it can return `ReflectionMethod|false`

Reply:
Noted, thanks, I'm aware about this change.